test/Interop/Cxx/value-witness-table/Inputs/basic.h

```diff
@@ -0,0 +1,36 @@
+#ifndef TEST_INTEROP_CXX_DESTRUCTOR_INPUTS_BASIC_H
```

Comment:
I'd suggest to rename these tests from "basic" to something specifically focused on destructors, since they are getting quite long and would likely only get longer as we add more variations when we implement class templates.

Reply:
I've renamed it to `CustomDestructorTypes`.